HelpSource/Classes/Git.schelp

```diff
@@ -32,6 +32,9 @@ subsection:: info
 METHOD:: remote, url
 returns:: url of the first remote that it finds.
 
+METHOD:: remoteAsHttpUrl
+returns:: remote url formatted for code::https:: request
```

Comment:
this could use some more explanation. without looking at the source, i've already forgotten what it means to format a URL for an https request.